src/main/java/org/takes/rq/form/package-info.java

```diff
@@ -32,11 +32,18 @@
  *
  * <p>All implementations of this interface must be immutable and thread-safe.
  *
+ * <p>Current implementations:
+ * {@link org.takes.rq.form.RqFormBase} base implementation;
+ * {@link org.takes.rq.form.RqFormFake} implementation that is useful for
+ * testing purposes. You can add your parameters in constructor;
+ * {@link org.takes.rq.form.RqFormSmart} decorator with extra features:
+ * Get single parameter and, if parameter not exists use default value or
+ * throw HTTP exception.
+ *
+ * <p>Also please look at usage examples:
+ *
```

Comment:
@mkordas I know. See my comment above (I asked if it will be ok just to embed some examples from test classes)